content/cloud-docs/users-teams-organizations/organizations.mdx

```diff
@@ -13,179 +13,188 @@ description: >-
-1. Lock all workspaces, and bring all current and pending runs to an end (by cancelling them or by waiting for them to finish). For your protection, Terraform Cloud won't change the name of an organization with runs in progress; locking workspaces ensures no new runs will start before you attempt to change the name.
-1. Rename the organization.
-1. Update anything that uses Terraform Cloud's API with the organization to use its new name. This includes Terraform's `remote` backend, the `tfe` Terraform provider, and any external API integrations.
-1. Unlock workspaces and resume normal operations.
+#### Tags
+
+Review a list of tags for all resources across the organization. When you delete a tag from this page, Terraform Cloud removes it from all resources.
 
-### Plan & Billing
+#### Teams
 
-The plan and billing page allows organization owners to upgrade to one of Terraform Cloud's paid plans, downgrade to a free plan, or begin a free trial of paid features. It also displays any invoices for previous plan payments.
+-> **Note:** Team management is a paid feature, available as part of the **Team** upgrade plan. Free organizations only include an owners team that can include up to five members. Refer to [Terraform Cloud pricing](https://www.hashicorp.com/products/terraform/pricing) for details.
 
-### Tags
+All users in an organization can access the **Teams** page, which displays a list of [teams][] within the organiation. This excludes [secret teams](/cloud-docs/users-teams-organizations/teams#team-visibility) where you are not a member. You can also view team membership and manage team API tokens.
 
-This page shows the list of tags used for all resources across the organization. Tags can be added directly to workspaces. Tags deleted here will be removed from all other resources, and can be deleted in bulk.
+Organization owners can also create and delete teams, manage team membership, and manage team API tokens. Remember that users must accept an invitation to the organization before you can add them to teams.
+
+[permissions-citation]: #intentionally-unused---keep-for-maintainers
 
-![Screenshot: the tags list, showing a list of currently used tags.](/img/docs/tag-management.png)
+#### Users
 
-### Teams
+Organization owners can invite Terraform Cloud users into the organization, cancel invitations, and remove existing members.
 
--> **Note:** Team management is a paid feature, available as part of the **Team** upgrade package. Free organizations only include an owners team, which can include up to five members. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing).
+The list of users has one tab for active users and one tab for invited users who have not yet accepted their invitations. For active users, the list includes usernames, email addresses, avatar icons, two-factor authentication status, and current team memberships. Use the **Search by username or email** field to filter these lists.
```

Comment:
```suggestion
The list of users is separated into one tab for active users and one tab for invited users who have not yet accepted their invitations. For active users, the list includes usernames, email addresses, avatar icons, two-factor authentication status, and current team memberships. Use the **Search by username or email** field to filter these lists.
```
Is the list of invited-but-not-accepted users worth mentioning in terms of contents?